pages/blog/june-2019-update.md

```diff
@@ -0,0 +1,43 @@
+---
+description: Here are the most significant changes in June.
+---
+
+# June 2019 Update
+
+**Olivier Tassinari**
+
+*July 8, 2019*
+
+Here are the most significant changes in June:
+
+- 💄 We have introduced a new [ButtonGroup](/components/buttons/#grouped-buttons) component.
+  ![ButtonGroup](/static/blog/june-2019-update/button-group.png)
+
+- 💄 We have introduced a new [Slider](/components/slider/) component.
```

Comment:
New™ 😉

Reply:
It should probably say that the slider has been overhauled and enhanced, and moved from lab to core.

Reply:
> We could make the TablePagination usable standalone

Several [pagination examples](https://www.ecosia.org/images?q=pagination) (including Bootstrap) are just a button-group. Perhaps we could start by adding a pagination example to the Button page? 🤷‍♂

Reply:
@mbrookes It's worth exploring 👍